src/main/scala/com/cognite/spark/datasource/EventsRelation.scala

```diff
@@ -121,6 +121,9 @@ class EventsRelation(config: RelationConfig)(@transient val sqlContext: SQLConte
       ()
     })
 
+  override def delete(rows: Seq[Row]): IO[Unit] =
+    deleteItems(config.auth, baseEventsURL(config.project), rows, config.maxRetries)
```

Comment:
Do we have batching somewhere to ensure that `rows` doesn't have too many items for a single request?

Reply:
Delete will be called on batches of rows, all writes using `save()` are batched (see DefaultSource: 215)